e_os.h

```diff
@@ -27,19 +27,16 @@
  * set this to a comma-separated list of 'random' device files to try out. By
  * default, we will try to read at least one of these files
  */
-#  if defined(__s390__)
-#   define DEVRANDOM "/dev/prandom","/dev/urandom","/dev/hwrng","/dev/random"
-#  else
-#   define DEVRANDOM "/dev/urandom","/dev/random","/dev/srandom"
-#  endif
+#  define DEVRANDOM "/dev/urandom", "/dev/random", "/dev/hwrng", "/dev/srandom"
```

Comment:
Why do you remove prandom? (I have no idea what prandom is)

Reply:
/dev/prandom is a linux device driver on s390, so I can read the source.
The issue with /dev/prandom is that it is seeded from /dev/urandom + time of boot
at the init time, the /dev/urandom is pretty poorly seeded.
From there everything is deterministic, until you pull 100KB out of the device,
then it will re-seed from /dev/urandom again.
@paulidale mentioned that to me, and I can only confirm.